### PR TITLE
Improves the lives of our poor clothes spriters by putting all vox through the femur breaker and making them fit in digitgrade clothes

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

TLDR for this entire section: Having to make a gazillion different variants of clothes just for the two vox that play is really really painful for spriters (me) and as a result most don't even make vox variants of clothes they make. So basically I'm respriting vox to fit in digitgrade clothes

But yes, as stated before, as someone who's made the unfortunate mistake of spriting clothes for all species on skyrat: God DAMN is making five (sometimes more) variants of the same clothing item disheartening. Oh boy! I finally got all these sprites done, and the ditigrade versions, and the teshari versions, finally I'm-

(The two entire other species with different body shapes in question)

Surely even someone who's not sprited before can see how this could be just AWFUL.

So the fact we have not one but two barely used vox species with entirely different body types despite being practically the same thing is just soul removing when seen from a sprite perspective. Digitgrade and even teshari clothes coverage is already spotty at best. Vox have barely any clothes support for obvious reasons following that.

So what to do? They don't need to be special, and we don't need to have two identical species that look a bit different. Instead we make them fit within snouted and digitgrade clothes like the rest of the species get to.

In better news, at least they get a resprite that looks half decent! How old even are the current vox sprites that this point?

## How This Contributes To The Skyrat Roleplay Experience

Uhh I kinda said everything I had to say above, basically it'll suck a gazillion times less for spriters making clothes because they won't have to make two entirely different vox variants of clothes that'll barely get used. Instead they'll just need to make the digitgrade versions.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
image
image
  
</details>

## Changelog

:cl:
qol: For the quality of life of all my homies out there that sprite clothes for skyrat, vox no longer use two different entirely unique body shapes that need to be sprited separately for.
del: Vox and Vox Primalis are now one single species, just vox again, because we didn't need two different identical species that looked slightly different and needed their own entirely unique body shapes of clothes.
image: New vox bodyparts made by me based roughly off the old bodyparts.
/:cl:

### CODE REVIEW

**Comment by Armyguy** [comment]
The fuck did you do too there poor heads?, they look like grays with fake beaks on.

**Comment by lpeapnni** [comment]
this shit sucks

**Comment by EerieDream** [comment]
Based keep it up

**Comment by ghost** [comment]
Blessed be. Kill the clothing files ending in _vox.dmi. Good riddance to them.

**Comment by Kepteyn** [comment]
While this does make sense, the new vox sprites look... off. While unfortunately inevitable for the body, the heads just don't look right - the proportions seem off, and doesn't look much like a vox. I would also consider what making their body purely bipedal does to the identity of the vox - the skulking, hunched posture is a significant part of the appeal, I reckon. Vox players may prefer to outright lose the ability to wear the dozens of clothes variants to keep the current bodyshape.

I'd also want to see what a vox primalis looks like - you said you've merged them, but you haven't shown what has become of the primalis variant. Are the options to make a three-eyed primalis vox just gone? Removing that option from players and forcing them to change their vox to be bird-like isn't very good.

**Comment by projectkepler-ru** [comment]
why do they look so scrungly

**Comment by Obliii** [comment]
No. Just no.

That is not a Vox anymore. That is a lizard with a beak. When will the line be drawn to stop removing good parts of the game and actually begin ADDING features to Skyrat?

**Comment by RegJackson** [comment]
> No. Just no.
> 
> That is not a Vox anymore. That is a lizard with a beak. When will the line be drawn to stop removing good parts of the game and actually begin ADDING features to Skyrat?

my brother in christ we just got
-synth wounds
-a fuckton of new sprites for digitgrades
-a massive rework of cargo that continues to make it more and more useful
-a new map (with another one down the pipeline)

and are currently in the process of getting
-a rework of the NRI's clothes
-taur sprites that actually look nice
-capes that look good as opposed to whatever the heck our cape sprites are right now
-gunshootening 3 the edition where paxil gives me specifically a fucked up evil plasma handcannon

everybody want sprite but nobody want to be spriter

**Comment by nikothedude** [comment]
Unique body types are cool, if a clothing item doesn't have a sprite, just make the vox not fit

**Comment by Nerev4r** [comment]
> No. Just no.
> 
> That is not a Vox anymore. That is a lizard with a beak. When will the line be drawn to stop removing good parts of the game and actually begin ADDING features to Skyrat?

....What?
We've been adding shit all month.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I feel like this is just going to cause a lot of discontent over something that's not ultimately going to make much of a difference in the long run.

We already don't really require spriters to add vox sprites (either variant) when they're adding new clothes, and so it's only really on a "do it if you feel like doing it" basis, so long as the clothes don't have the missing texture appearance on them when applied, I don't think it's really an issue.

I don't really want to add more bodytypes moving forward, but considering we already have all of those sprites for them, I don't see a point in removing them for something that's not mandatory.

I'm not closing this yet, but I'm not very likely to merge this one. I feel like it's only going to encourage people to play Vox even less.

**Comment by CliffracerX** [comment]
A version of this that tweaked normal Vox to match the digi bodytype whilst leaving their heads alone would work pretty well given they're already not far off one of the old incarnations of digitigrade bodies as it is, but nuking the Primalis alongside isn't great.  Primalis and Tesh players have long been used to not getting the full extent of clothing options that normal critters have, and spriters have been largely fine with leaving it be to focus on what the majority of the playerbase use; when something really good comes along that the alt-bodytype playerbase wants, one of 'em almost always picks up the slack to port it.

Much like when Tesh had their hairstyles nuked, this isn't great; Skyrat's humongous library of options for creating characters is one of its greatest strengths, and anything that hurts it is a damn shame.

**Comment by GoldenAlpharex** [comment]
I forgot to come back and close this, here we go.
